go.mod

```diff
@@ -131,3 +132,5 @@ require (
 replace github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.10.0
 
 replace istio.io/api => istio.io/api v0.0.0-20211103171850-665ed2b92d52
+
+replace mosn.io/proxy-wasm-go-host => ../pp
```

Comment:
this is why build is broken for now. I'll migrate it once proxy-wasm-go-host is merged